Tasks/AzureRmWebAppDeployment/FindInstalledMSDeploy.ps1

```diff
@@ -0,0 +1,146 @@
+﻿
+function Get-MSDeployOnTargetMachine
+{
+    try
+    {
+        $MSDeployPath, $MSDeployVersion = Locate-HighestVersionMsDeployPackage
+        $MSDeployVersionNumber = [decimal] $MSDeployVersion
+    }
+    catch [System.Exception]
+    {
+        Write-Verbose ("Failed to get MS Deploy location with exception: " + $_.Exception.Message)
+        $MSDeployVersionNumber = 0
+    }
+
+    if ($MSDeployPath -eq $null)
+    {
+        throw  "Unable to find the location of MS Deploy from registry on machine $env:COMPUTERNAME"
```

Comment:
Should it be throw or verbose message, as we want to continue execution with pre bundled msdeploy